src/diffusers/loaders/lora_pipeline.py

```diff
@@ -2667,6 +2667,314 @@ def unfuse_lora(self, components: List[str] = ["transformer", "text_encoder"], *
         super().unfuse_lora(components=components)
 
 
+class Mochi1LoraLoaderMixin(LoraBaseMixin):
```

Comment:
A copy-paste of the Cog LoRA loader classes, indicated by the "Copied from ..." comments.